distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -147,6 +147,9 @@ ALERT;SMA Energy Meter: The Thing configuration has changed and now also require
 ALERT;Transformations: NULL and UNDEF item states are now passed to any defined transformation.
 ALERT;Transformations-MAP: "-" entry defined in a MAP file is no more used by sitemap UIs when the item state is NULL or UNDEF. You can now define "NULL" and "UNDEF" entries to map these 2 particular states or rely as before on the definition of a default mapping to map anything that has no entry in your file including NULL and UNDEF.
 
+[4.3.0]
+ALERT;Pentair Binding: EasyTouch thing has been renamed to more generic Controller and all channels have been orgnanized into groups. You will need to reconfigure your setup to the new thing structure.  Auto discovery of things has also been added.
```

Comment:
```suggestion
ALERT;Pentair Binding: EasyTouch thing has been renamed to more generic Controller and all channels have been organized into groups. You will need to reconfigure your setup to the new thing structure. Auto discovery of things has also been added.
```

Reply:
Perhaps the last sentence about auto discovery can be dropped, since that is not a breaking change?

Reply:
It seems I cannot unresolve this comment, but the typo is still there.